com.unity.ml-agents/Runtime/Analytics/InferenceAnalytics.cs

```diff
@@ -20,6 +20,7 @@ internal class InferenceAnalytics
     {
         const string k_VendorKey = "unity.ml-agents";
         const string k_EventName = "ml_agents_inferencemodelset";
+        const int k_EventVersion = 1;
```

Comment:
I think we'll need to use version 2 here, waiting to hear back from analytics team.